### ISSUE TITLE
Omg now my main account is working :D

### ISSUE THREAD

**Issue by AlexCtasy** [finndev/PokeBuddy#1026]
Okay guys, now my account, the main is working. So happy.. xaoxoaxo is not a perma ban xD

**Comment by RoseCorpse** [comment]
omg, let me check mine

**Comment by iPetrov2** [comment]
@AlexCtasy i think i survived the Ban Wave.

**Comment by RoseCorpse** [comment]
nope, still dead

**Comment by AlexCtasy** [comment]
@RoseCorpse Still showing u message of file data? Strange thing, maybe i got unban cause I donate coins when I started and today I donated again xD

**Comment by dragonite00123** [comment]
What ban is this? is this the outofrange bullsh*t?

**Comment by AlexCtasy** [comment]
@dragonite00123 Yes dude, i was with that out of range and now, 20 min ago, I free :D Now i can use any pokespot with my main account.

**Comment by RoseCorpse** [comment]
it shows the "servers are down, sorry for the inconvenience" just in my main char, other no botting chars are normal.

**Comment by ISeishiro** [comment]
my main account also is working now
i can activate pokestops. damn im so happy, i thought ill lose all the rare ive caught.

**Comment by AlexCtasy** [comment]
@RoseCorpse Try again and try again, log out and log in and then u gonna log ur main

**Comment by AlexCtasy** [comment]
@iseishiro That ban is for like 12hous and u free..

**Comment by RoseCorpse** [comment]
I have no hope :sob:

**Comment by RoseCorpse** [comment]
now "failed to get game data from server"

**Comment by AlexCtasy** [comment]
@RoseCorpse Well btw are you botting another char with necrobot? go do it and solve all sh1t.

**Comment by dragonite00123** [comment]
@AlexCtasy  me also, out of range ban is really frustrating last night.. haha..

**Comment by AlexCtasy** [comment]
@dragonite00123 U still banned of out of range? Or now u free to use pokestops?

**Comment by Artillery101** [comment]
How do you log out?

**Comment by AlexCtasy** [comment]
From options in game :S

**Comment by Artillery101** [comment]
Can't get in.

**Comment by ISeishiro** [comment]
@AlexCtasy yeah i think its really 12 hrs soft ban now. xD

if you guys still can access ur account, then you still have hope. just wait for it and be patient it will work again for you guys. goodluck!

**Comment by Artillery101** [comment]
From when?

**Comment by RoseCorpse** [comment]
I just erase game data to "log off"

**Comment by Artillery101** [comment]
Delete the app?

**Comment by RoseCorpse** [comment]
no, just erase game data

**Comment by RoseCorpse** [comment]
and that necrobot, I cant find where to download

**Comment by GObang84** [comment]
It's true, i have [Farm Pokestops] OutOfRange -> Received XP: 0 issue and i'm able to spin pokestops again. I'm not sure about 12hours but i wake up this morning and it's working. (more than 12hours from the time of softbanned.)

**Comment by Artillery101** [comment]
How do you delete anything if you can't get in?

**Comment by AlexCtasy** [comment]
Yes, is like 12h or more now the softban, i dont know why some people get the message of file data, but if u can access ur account till u are out of range u have hope to get unban.. If you cannot so is the end.

**Comment by RoseCorpse** [comment]
configuration - apps - pokemon go - detele cache and data

**Comment by Artillery101** [comment]
If you delete data, what are you left with?

**Comment by AlexCtasy** [comment]
Nothing dude, just delete it and try again to log in.

**Comment by Artillery101** [comment]
Or wait a few more hours?

**Comment by AlexCtasy** [comment]
Well btw guys, i hope niantic is just bannning for use bots. Because Ill non stop to use fake gps, here in my country is not safe to go in the street with an iphone to play a game, so if niantic decide to ban fake gps I really dont care to get banned, btw I 100% sure that the are banning just for bots.

**Comment by RoseCorpse** [comment]
you just delete account data in game, nothing else

**Comment by AlexCtasy** [comment]
@artillery101 How many hours are u waiting?

**Comment by Artillery101** [comment]
Not sure.  I checked this morning around 6-30 and was locked out, so hoping by tonight (ie around 3 1/2 hrs) it will be back.

**Comment by RoseCorpse** [comment]
I just want a miracle, I had 142 pokemons in my acc :sob: :sob: :sob:

**Comment by Artillery101** [comment]
Bugger

**Comment by AlexCtasy** [comment]
@RoseCorpse Lol i have more than you in my main account. Not full CP.. But btw.. If they start to ban fake gps, I sure I delete the app from my phone. I prefer dont play a pogo that lose my iphone 6s.

**Comment by RoseCorpse** [comment]
this is Maduro's fault.

**Comment by MJH94** [comment]
@AlexCtasy dont use pokebuddy tho. my main acc started working a few hours ago and necrobot is still way to go

**Comment by RoseCorpse** [comment]
can someone please link me that damn necrobot?

**Comment by MJH94** [comment]
#998

**Comment by AlexCtasy** [comment]
@mjh94 btw ur main account is working now and u are botting with ur main using necro? or u stop to bot with ur main?

**Comment by RoseCorpse** [comment]
nope, nothing there.

**Comment by AlexCtasy** [comment]
I guess the admin removed the file from the page, cause he will bot with that xoaoxaox no one need to know xD

**Comment by RoseCorpse** [comment]
@AlexCtasy can you pleeeeease upload necrobot to mega and link me? D:

**Comment by MJH94** [comment]
botting main with necro

**Comment by AlexCtasy** [comment]
@MJH94 how u do it xoaoxa, I tried to bot my main right now with necrobot and i got again the out range banned, So i will keep botting the another account that is lv 23 and my main ill wait again for more than 12 hors xoaxoaox

**Comment by MJH94** [comment]
it just started working for me a few hours ago. nothing different. settings all the same

**Comment by AlexCtasy** [comment]
@MJH94 well maybe cause i tried to use my main account in my phone with the facking gps on in the mobile, and also using the main account botting in necro, and cause that i got again the out range ban... I dont really know.

@RoseCorpse Ill try to upload it and i link u

**Comment by RoseCorpse** [comment]
@AlexCtasy thanks so so much.

**Comment by MJH94** [comment]
yeah i havent even tried it in my phone

**Comment by MJH94** [comment]
im waiting until i get to lvl 40

**Comment by AlexCtasy** [comment]
@mjh94 MY fault i guess was that I was botting with the main account while i logged it also in my phone, well was my mistake, same like 12hours ago, get again the out range banned, another 12 hours more, btw i dont care, i use this account lv23 to keep botting. Btw why get lv 40.. If i get lv 40 i bored and unistall the game xoaoxa.

**Comment by RoseCorpse** [comment]
(comment in spanish just for AlexCtasy) 

necesitoooo seguir jugando!! me dijeron que me quitaron un gym y de paso es un tipo que conozco con el que tuve verguero en clash of clans, quiero sequir jugando para restregarle en la cara a mi dragonite 3000+ y reirme en su cara U_____u

**Comment by AlexCtasy** [comment]
@RoseCorpse LOL jaja, cuantos pokemones +3000 cp tenias en tu cuenta principal? jajaj dime eso..

Hay como mandarte el link por privado aqui?

**Comment by RoseCorpse** [comment]
tenia solo un dragonite que me costo mucho evolucionar con ese cp, tenia muchos mas entre 2000-2500cp, de verdad no se si se puede por aca, dejame ver

**Comment by AlexCtasy** [comment]
@MJH94 This sh1t is so strange dude, now i tried again my account in the phone and is working again and i allowed to use pokestops again, tell me something? What u think that is better i just use my main account PTC in the necrobot and do not log it in the phone? to avoid again the out range ban?

**Comment by RoseCorpse** [comment]
ya se como, entra en esta pagina http://foro.sro-latino.com/foro.php al final hay un chat, ahi estoy, pregunta por mi

**Comment by AlexCtasy** [comment]
@MJH94 Are u here dude?

**Comment by willhoo** [comment]
who can send me a link for download necrobot? thank you

**Comment by AlexCtasy** [comment]
@MJH94 xaoxoa now i using my main account with like 8 eggs for xp and 4 insence + necrobot xaoxaox, let me tell you my exp is going up like crazy. xD

**Comment by Kokainium** [comment]
I dont know what u guys are doing. But i have noget ban issues ? :o just have to change ip sometimes:)

**Comment by Kokainium** [comment]
No*

**Comment by pixelpeeker** [comment]
hey @AlexCtasy can you send me the necrobot download link too? thank you :smile:

**Comment by Artillery101** [comment]
I'm still locked out.  :-(.  Might just delete the lot and find something
else.

**Comment by pixelpeeker** [comment]
![screenshot_34](https://cloud.githubusercontent.com/assets/20956338/17642672/a00d97f8-6178-11e6-91c8-4c50fe822eee.png)
i just found the necro, but this thing is showing up upon the opening. any solutions?

**Comment by Artillery101** [comment]
I've uninstalled the app, had a new system upgrade, reinstalled the app and
still locked out.  Can't get into it to log out.

**Comment by Hyperhk** [comment]
@AlexCtasy have u ever had "failed to get game data from server"?

**Comment by Hyperhk** [comment]
@pixelpeeker necrobot is over they have stopped providing services

**Comment by AlexCtasy** [comment]
@Hyperhk No, I was banned like 12hs but i was able to log my account, use all features and just cannot use the pokespots "try again later", then I was unbanned and tried again to bot, banned again for 1 hour, and now I still boting with necro like 8hours ago and all is fine, i really dont know why people are getting that message "failed to get data from server", dont know what u did to get that message.

**Comment by AlexCtasy** [comment]
@Hyperhk I using right now with my main account necrobot, 8 hours ago using it, all is fine.

**Comment by Artillery101** [comment]
I don't know either.

**Comment by Hyperhk** [comment]
@AlexCtasy when u get taht message, that's a perma ban, what you got was a softban which is not big deal.

**Comment by Artillery101** [comment]
Over 24 hours and still can't get in. Think it's time I deleted the whole
lot and find something else to do, like real life. :-)

**Comment by novalbert** [comment]
@Hyperhk Is it really perma ban? Is there any solution??
I see more people outside still dont believe it's a perma ban

**Comment by Artillery101** [comment]
I still can't get in.  Been 2 1/2 days now.
